marketplace/PREREQUISITES.md

```diff
@@ -14,13 +14,19 @@ solution automatically.
 Enable the Kubernetes API as well, since the Datashare API is deployed to Cloud Run on Anthos Kubernetes by default.  
 
 ## Enable the APIs
-Note you can only enable the Commerce API (cloudcommerceprocurement.googleapis.com) if your project has been permissioned to the Partner Portal, otherwise you will need to enable this permission after you are provided with access to Partner Portal.
+Note you can only enable the Cloud Commerce Producer API (cloudcommerceproducer.googleapis.com) or the Cloud Commerce Partner Procurement API (cloudcommerceprocurement.googleapis.com) if your project has been permissioned to the Producer or Partner Portal, otherwise you will need to enable this permission after you are provided with access to Producer Portal Partner Portal.
```

Comment:
Can we add a link to redirect users to the Marketplace partner team if their project has not been enabled/permissioned yet?